### PR TITLE
Add quick_model option + print time

### PR DESCRIPTION
**Pull request by maxentile**
- Prints total time elapsed after each stage
- Adds "quick_model" option that uses cheaper features and fewer tICs
- Renders .png instead of .jpg to avoid mysterious matplotlib backend error on hal

### CODE REVIEW

File: pipeline.py [hunk 1]
```diff
@@ -83,11 +83,15 @@ def run_pipeline(fnames,
...
         feat.add_residue_mindist(residue_pairs = respairs_that_changed, scheme = scheme)
 
-    print('Num features: ', len(feat.describe()))
+    n_features = len(feat.describe())
+    print('Num features: ', n_features)
 
     # do featurization + tICA by streaming over size-1000 "chunks"
     source = pyemma.coordinates.source(fnames, features = feat)
     source_full = pyemma.coordinates.source(fnames, top = top)
+    if quick_model:
```
Comment (jchodera) [inline pipeline.py:92]: Good idea!

**Comment by sonyahanson** [comment]
Thanks!

**Comment by sonyahanson** [comment]
Make sure you aren't reverting any changes here. I just checked out this branch, and got the old 'too many values too unpack' error when trying to run `test_report_generation.py`.

**Comment by sonyahanson** [comment]
Nevermind. I think that was somehow my fault, you haven't changed `test_report_generation.py` in your commits.

**Pull request closed without merging**